react/src/views/BoxCreate/BoxCreateView.tsx

```diff
@@ -105,14 +106,14 @@ function BoxCreateView() {
   // Check the QR Code
   useEffect(() => {
     if (qrCodeExists.data?.qrExists === false) {
-      notificationVar({
+      createToast({
```

Comment:
Why is `triggerError` not used here?

Reply:
Clarified: we don't want the side effect of console error and logging to sentry, *only* the toast displayed